packages/front-end/components/Experiment/ImportExperimentList.tsx

```diff
@@ -41,6 +42,12 @@ const ImportExperimentList: FC<{
     existing: Record<string, string>;
   }>(`/experiments/import/${importId}`);
 
+  const dataSourceData = useApi<{
+    settings: DataSourceSettings;
+  }>(`/datasource/${data?.experiments.datasource}`);
```

Comment:
You don't need an API call for this.   If you look further down around line 80, you can see 
```ts
const datasource = getDatasourceById(data.experiments.datasource);
```
Then you can just add this line after that:

```ts
const isExperimentName = datasource?.settings?.queries?.exposure?.[0]?.hasNameCol;
```

Also, notice how I did `exposure?.[0]` instead of `exposure[0]`.  That will stop it from throwing an error if `exposure` is null or undefined.

Reply:
Resolved by new changes